lib/winston-logstash-udp.js

```diff
@@ -46,7 +46,13 @@ util.inherits(LogstashUDP, winston.Transport);
 winston.transports.LogstashUDP = LogstashUDP;
 
 LogstashUDP.prototype.connect = function() {
+    var self = this;
     this.client = dgram.createSocket('udp4');
+
+    this.client.on('error', function(err) {
+        // Skipping DNS errors
+        self.emit('error', err);
```

Comment:
According to the thread you mentioned, to skip the DNS errors, this should be wrapped in a check for the `ENOTFOUND` error code.

```javascript
if (err.code !== 'ENOTFOUND')
```